src/config/index.js

```diff
@@ -140,9 +140,12 @@ const store = new Confidence.Store({
                 }
             }
         },
-        weather      : {
+        weather        : {
             openWeatherApiKey : { $env : 'OPEN_WEATHER_API_KEY' },
             LocationIQApiKey  : { $env : 'LOCATION_IQ_API_KEY' }
+        },
+        isthereanydeal : {
+            apiKey : { $env : 'ISTHEREANYDEAL_API_KEY' },
         }
```

Comment:
Should be under a `tool` attribute

```suggestion
        tool : {
            isthereanydeal : {
                apiKey : { $env : 'ISTHEREANYDEAL_API_KEY' }
            }
        }
```